src/resources/CheckoutInfo.ts

```diff
@@ -58,6 +58,10 @@ export interface CheckoutInfoItem {
         enabled?: boolean;
         threshold?: AmountWithCurrency[];
     };
+    installmentsConfiguration: {
```

Comment:
Note: API uses `installementsConfiguration` for now but they will fix the typo in the next releases 🙂